src/extract.js

```diff
@@ -50,11 +50,23 @@ exports.TranslationReference = class TranslationReference {
 };
 
 function preprocessScriptTags(data, type) {
+  if (type !== 'vue') {
+    return {};
+  }
+
   const vueFile = vueCompiler.parse({ source: data, needMap: false });
+  if (!vueFile.script) {
+    return {};
+  }
```

Comment:
no :) if you do this you will hide the fact that it failed. 

Raise (or return null but it's not as good), check it and continue the loop in the calling code.

Reply:
I addressed this comment and added a test at 57af6f0